internal/cluster/distribution/eks/ekscluster/eks.go

```diff
@@ -232,6 +233,11 @@ func (a *NodePool) Validate(npName string) error {
 		return err
 	}
 
+	// ---- [ Node pool name validation ] ---- //
+	if err := pkgCommon.ValidateNodePoolName(npName); err != nil {
+		return err
+	}
+
```

Comment:
**Optional**: I personally would start with the name validation, but this is entirely optional, just a thought.

Reply:
I put it there first but then thought it would look better together with the label validation at the end, but definitely makes more sense in to start with that 🙂

Reply:
I meant just before the label validation, not at the start of the function, but it is good as it is (whichever it is right now), not worth cycles over it :) .